### DISCUSSION TITLE
/dev/nvidia0 on WSL2

### DISCUSSION THREAD

**Discussion by IvensaMDH** [microsoft/WSL#8060] [Q&A]
Hi,

Currently getting "Could not open device '/dev/nvidia0' for reading and writing." when trying to launch a HW accelerated decoding job. There is no /dev/nvidia0 device present on my WSL2 - is there any way to access this?

Thanks

**Comment by markgeejw** [comment]
Hello, my current workflow requires using Deepstream's nvv4l2decoder for hardware-accelerated decoding and it complains that /dev/nvidia0 is not found. Is it possible to get it to work with WSL2?
